A32NX/html_ui/Pages/A32NX_Core/A32NX_GPWS.js

```diff
@@ -88,6 +88,7 @@ class A32NX_GPWS {
             this.Mode5Code = 0;
 
             this.Mode3MaxBaroAlt = NaN;
+            this.Mode4MaxRAAlt = NaN;
```

Comment:
The max RA alt gained is not supposed to be deleted if the aircraft is above the threshold (it should be saved until landing). If the aircraft is below the threshold though (i.e. on the ground), resetting it is o.k, so I would just add a check if RA is below 10 ft.
See here as a reference from the FCOM:
![grafik](https://user-images.githubusercontent.com/39596827/108513194-8a090180-72c2-11eb-8e3f-21582ecb6f82.png)

Reply:
Thanks for the reference.

I've updated it to reset only on RA lower than 10 or `onGround`. I'm not sure how reliable the `SIM ON GROUND`, hence the additional RA comparison.